packages/adapters/storage/src/lib.rs

```diff
@@ -506,16 +506,19 @@ mod tests {
         storage: impl services::block_importer::port::Storage,
         range: RangeInclusive<u32>,
     ) {
-        let blocks = range
-            .clone()
-            .map(|height| CompressedFuelBlock {
-                height,
-                data: nonempty![height as u8],
-            })
-            .collect_nonempty()
-            .expect("shouldn't be empty");
-
-        storage.insert_blocks(blocks).await.unwrap();
+        let chunk_size = 10_000;
+        for chunk in range.clone().collect::<Vec<_>>().chunks(chunk_size) {
+            let blocks = chunk
+                .iter()
+                .map(|&height| CompressedFuelBlock {
```

Comment:
```suggestion
        for chunk in range.chunks(chunk_size).into_iter() {
            let blocks = chunk
                .map(|height| CompressedFuelBlock {
```

Reply:
The chunking here is probably for managing memory usage, but we should add a comment explaining why we chunk (since there is chunking behind the scenes in the storage impl that is required by the db driver)